src/cascadia/TerminalControl/TermControl.h

```diff
@@ -81,6 +82,22 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         hstring _uri;
     };
 
+    struct NoticeEventArgs :
+        public NoticeEventArgsT<NoticeEventArgs>
+    {
+    public:
+        NoticeEventArgs(NoticeLevel level, hstring message) :
+            _level(level),
+            _message(message) {}

```

Comment:
Nit: you can either use `const&` for message or `std::move()` it on initialization to save a refcount.